src/App.tsx

```diff
@@ -8,7 +8,7 @@ import styles from './styles.module.css'
 
 function App() {
   const [anchorId, setAnchorId] = useState('button')
-  const [isDarkOpen, setIsDarkOpen] = useState(false)
+  const [isDarkOpen, setIsDarkOpen] = useState(true)
```

Comment:
I had to set the default state to open to have the access to the shadow dom element